### ISSUE TITLE
Smudgeplot and genomescope

### ISSUE THREAD

**Issue by nhartwic** [KamilSJaron/smudgeplot#114]
I mostly work with plants. I've started playing with smudgeplot more seriously lately. I'm getting a lot of cases where smudgeplot results seem misleading. Here is one...

![transformed_linear_plot](https://user-images.githubusercontent.com/13526494/232855306-52cdec7f-634b-45df-ba4a-c8908d45d389.png)
![smudgeplot_smudgeplot](https://user-images.githubusercontent.com/13526494/232855368-59650d69-f924-4e0e-96a7-6885d560a72f.png)

...In addition to genomescope and smudgeplot results, I also have a genome assembly for this exact genotype using ONT and Illumina data. This 274 megabase assembly has a QV score of 50 with respect to the Illumina reads processed by genomescope/smudgeplot and has good BUSCO scores (99% complete 4% duplication). In total, we have 33.5 gigabases of illumina reads.

Assuming this plant is diploid, that means we should have a 1N coverage of about 60x (53x accounting for kmer size) based on assembly size and reads. Genomescope reports a 1N coverage of about 40x. Smudgeplot reports a 1N coverage of about 80x. The assembled genome size and genomescopes reported size are pretty close if you squint at it. I can force smudgeplot to use a 1N coverage of 41, in which case it reports a dominant AABB peak. But setting genomescope to "-p 4" produces much worse assembly size estimates.

Basically, I have a mess of statistics from various analysis that don't seem mutually consistent at first glance and would love some input on how this smudgeplot can be interpreted. I have similar results for several dozen more datasets I need to figure out as well.

**Comment by nhartwic** [comment]
Thanks for the reply.

> Long story short - smudgeplot estimate the coverage to be 2x higher than it really is.

Well ya. I guess my real question is if we can say anything about the fact that the het kmer signal seems to be dominated by an AABB region of the genome despite the genome seeming to be otherwise diploid and relatively homozygous. Do we think these het-kmers are the result of ancestral WGD that just haven't been quite squeezed out of the genomes yet? Or are these just a result of genomic kmers selected at random sometimes being hetkmers? Or something else entirely?

> Re coverage and genome estimates.

I'm not really concerned about the difference in size estimate between genomescope and my assembled genome size. It is frankly pretty normal, especially when we start with green tissue which always includes a bunch of chloroplast DNA. I apologize if my language didn't make that clear in my previous comment. I'm used to genomescope size estimates being a bit off relative to haploid assembly size due to some mix of contamination, non-genomic DNA, or collapsed repeats.

**Comment by KamilSJaron** [comment]
Where do you get the "relatively homozygous"? To me it's nearly completely homozygous.

Nearly all (not really all) of the hetkmers here are paralogs (AABB is extremly unlikely to be representing two heterozygous sites with exactly the same two haplotypes, a lot more parsimonious is some ancestral duplication of "not yet so diverged" region) - I think that's you you mean by result of WGD, right?

Good about the genome size :-)

**Comment by nhartwic** [comment]
> but I think you got all that, that's you you mean by result of WGD, right?

Ya. Thanks again for the input.  I think we can close this issue.
